object-destructuring/exercise-1/exercise.js

```diff
@@ -1,13 +1,20 @@
-const personOne = {
-  name: "Popeye",
-  age: 34,
-  favouriteFood: "Spinach",
-};
+// const personOne = {
+//   name: "Popeye",
+//   age: 34,
+//   favouriteFood: "Spinach",
+// };
 
-function introduceYourself(___________________________) {
+// function introduceYourself(___________________________) {
+//   console.log(
+//     `Hello, my name is ${name}. I am ${age} years old and my favourite food is ${favouriteFood}.`
+//   );
+// }
+
+// introduceYourself(personOne);
+function introduceYourself({ name, age, favouriteFood }) {
```

Comment:
Please, avoid having commented out code in your PRs. I would just delete the block, but if I know that I might need it in the future, I would write down previous commit hash for the reference.